modules/tinymce/src/plugins/help/main/ts/ui/PluginsTab.ts

```diff
@@ -63,10 +63,9 @@ const tab = (editor: Editor): Dialog.TabSpec & { name: string } => {
   const getPluginKeys = (editor: Editor) => {
     const keys = Obj.keys(editor.plugins);
     const forcedPlugins = Options.getForcedPlugins(editor);
+    const hiddenPlugins = Type.isUndefined(forcedPlugins) ? [ 'onboarding' ] : forcedPlugins.concat([ 'onboarding' ] );
```

Comment:
I was thinking maybe we find a more definitive way to make sure it's our plugin before excluding it... but I doubt it matters too much. Not worth the effort.

Reply:
We can fix that bug when and if it happens. Unlikely. Maybe we figure out how to do the on/off switch proper in dispenser before then, then we could get rid of this code and use forced_plugins.